backend/src/main/java/instantPolls/model/QuestionMessage.java

```diff
@@ -0,0 +1,61 @@
+package instantPolls.model;
+
+import java.util.ArrayList;
+
+public class QuestionMessage {
```

Comment:
COnsider implementing builder pattern for this class. Some calls above would be so much nicer!